curvefs/proto/topology.proto

```diff
@@ -38,7 +38,7 @@ enum TopoStatusCode {
     TOPO_IP_PORT_DUPLICATED = 14;
     TOPO_NAME_DUPLICATED = 15;
     TOPO_CREATE_COPYSET_ON_METASERVER_FAIL = 16;
-    TOPO_CANNOT_REMOVE_NOT_RETIRED = 17;
+    TOPO_CANNOT_REMOVE_NOT_OFFLINE = 17;
```

Comment:
do we have `RETIRED` state for metaserver now?

Reply:
> do we have `RETIRED` state for metaserver now?

Metaserver does't have this state.